platform/vs/tests/breakout/test_breakout_cli.py

```diff
@@ -0,0 +1,166 @@
+from swsscommon import swsscommon
+import time
+import os
+import json
+import ast
+import pytest
+import collections
+
+@pytest.mark.usefixtures('dpb_setup_fixture')
+class TestBreakoutCli(object):
+    def setup_db(self, dvs):
+        self.cdb = swsscommon.DBConnector(4, dvs.redis_sock, 0)
+
+    def read_Json(self, dvs):
+        test_dir = os.path.dirname(os.path.realpath(__file__))
+        sample_output_file = os.path.join(test_dir, 'sample_output', 'sample_new_port_config.json')
+        fh = open(sample_output_file, 'rb')
+        fh_data = json.load(fh)
+        fh.close()
+
+        if not fh_data:
+            return False
+        expected = ast.literal_eval(json.dumps(fh_data))
+        return expected
+
+    def breakout(self, dvs, interface, brkout_mode):
+        (exitcode, result) = dvs.runcmd("config interface breakout {} {} -y".format(interface, brkout_mode))
+
+        if result.strip("\n")[0] == "[ERROR] Breakout feature is not available without platform.json file" :
+            pytest.skip("**** This test is not needed ****")
+        root_dir = os.path.dirname('/')
```

Comment:
Is this used anywhere?

Reply:
As we already have the platform.json file in vs, so it has not been exercised yet. But Good to have. We have this check in the [dev code](https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L2130)

Reply:
Oh the check looks good! I was talking about the `root_dir` variable.

Reply:
ooh okay. my bad. Nope, during breakout operation,  `new_port_config.json` file gets [created](https://github.com/Azure/sonic-utilities/blob/master/config/main.py#L2197). Here, we are utilizing the file so that we can compare the result of the breakout mode output to the expected one.